### PR TITLE
Add ability to edit debts

### PR DESCRIPTION
**Pull request by tudor-gala**

### CODE REVIEW

**Comment by Phrohdoh** [comment]
How is update different from edit (semantically)? Why have two methods?

File: app/Http/routes.php [hunk 1]
```diff
@@ -84,6 +84,8 @@
 
         // Debt
         Route::get('/people/{people}/debt/add', ['as' => '.debt.add', 'uses' => 'PeopleController@addDebt']);
+        Route::get('/people/{people}/debt/{debtId}/edit', ['as' => '.debt.edit', 'uses' => 'PeopleController@editDebt']);
+        Route::post('/people/{people}/debt/{debtIt}/update', ['as' => '.debt.update', 'uses' => 'PeopleController@updateDebt']);
```
Comment (Phrohdoh) [inline app/Http/routes.php]: `debtIt`?

**Comment by tudor-gala** [comment]
Conventionally, Laravel uses edit when referring to the action of showing the user the edit form for a certain resource, and update when referring to the POST action that actually updates that resource with new data.

File: app/Http/routes.php (hunk 1 above)
Comment (tudor-gala) [inline app/Http/routes.php]: Good catch! That was a typo, fixed it now

File: app/Http/Controllers/PeopleController.php [hunk 2]
```diff
@@ -1540,6 +1556,38 @@ public function storeDebt(Request $request, $contactId)
...
+            return redirect()->route('people.index');
+        }
+        
+        $debt->in_debt = $request->input('in-debt');
+        $debt->amount = $request->input('amount');
+        $debt->reason = $request->input('reason');
+
+        $debt->save();
+
+        $contact->logEvent('debt', $debt->id, 'edit');
```
Comment (Phrohdoh) [inline app/Http/Controllers/PeopleController.php]: In https://github.com/tudor-gala/monica/blob/e1e08d349210df7be2937785a53b99a8c3dcba16/app/Http/Controllers/PeopleController.php#L150 this is logged as `update`, not `edit`.

IMO this should be `update` given the above.

**Comment by Phrohdoh** [comment]
LGTM with my nit addressed 👍

File: app/Http/Controllers/PeopleController.php (hunk 2 above)
Comment (djaiss) [inline app/Http/Controllers/PeopleController.php]: Indeed, this should be `update`. If it's `edit` it's gonna raise an error on the Dashboard.

Comment (tudor-gala) [inline app/Http/Controllers/PeopleController.php]: Fixed this. Also changed the docblock for logEvent to correctly indicate that 'add', 'update', or 'delete' should be used :-)

File: resources/lang/en/people.php [hunk 3]
```diff
@@ -276,5 +276,7 @@
     'debt_add_amount' => 'the sum of',
     'debt_add_reason' => 'for the following reason (optional)',
     'debt_add_add_cta' => 'Add debt',
+    'debt_edit_update_cta' => 'Update debt',
```
Comment (djaiss) [inline resources/lang/en/people.php]: I'm going to be super annoying, but you've introduced new keys, which is not a problem - but they new keys should also be put on the other translation files as well (you don't need to translate them, just paste them as is).

Comment (tudor-gala) [inline resources/lang/en/people.php]: No problem, you're not annoying at all, I actually appreciate you taking the time to explain these things so I'll know in the future!
I've added the missing translations in a new commit.

**Comment by djaiss** [comment]
This is amazing. Thanks for your work.

**Pull request merged**